docs/reference/filters.md

```diff
@@ -2385,3 +2385,27 @@ pr: Path("/products/:productId")
 consistentHashKey("${request.header.Authorization}")
 consistentHashKey("${request.source}") // same as the default key
 ```
+
+## consistentHashBalanceFactor
+
+This filter sets the balance factor used to prevent a single backend endpoint from being overloaded.
+The number of in-flight requests for an endpoint can be no higher than `(average-in-flight-requests * balanceFactor) + 1`.
+This is helpful in the case where certain keys are very popular and threaten to overload the endpoint they are mapped to.
+[Further Details](https://ai.googleblog.com/2017/04/consistent-hashing-with-bounded-loads.html).
```

Comment:
Additionally it may also link to https://github.com/zalando/skipper/blob/master/docs/reference/backends.md#load-balancer-backend and that page may link back to this filter like it does for `consistentHashKey`